src/vuestic-theme/vuestic-components/va-input/VaInput.vue

```diff
@@ -187,6 +210,17 @@ export default {
     },
   },
   methods: {
+    adjustHeight () {
+      const { autosize } = this.$props
+      if (!autosize || !this.isTextarea) return
+
+      const minRows = this.minRows || 1
+      const maxRows = this.maxRows || Number.MAX_SAFE_INTEGER
+      const textareaStyles = calculateNodeHeight(this.$refs.input, false, minRows, maxRows)
+
+      Object.assign(this.$refs.input.style, textareaStyles)
```

Comment:
Some explanation on why we modify dom directly instead on relying on reactivity would be nice to have.